app/infrastructure/database/local/category.yml

```diff
@@ -2,30 +2,33 @@
 - ud_category_id: 240
```

Comment:
Do you still want to leave these yml file on version control?
And what's your decision about storing data beforehead or not?